src/meta/play-meta.js

```diff
@@ -129,6 +130,31 @@ export const plays = [
     blog: '',
     video: ''
   }, {
+    id: 'pl-react-todo-app',
+    name: 'React Todo App',
+    description: 'It is a simple Todo App which keeps track of your regular work',
+    component: () => { return <ReactTodoApp /> },
+    path: '/plays/react-todo-app',
+    level: 'Beginner',
+    tags: 'ReactHooks, JavaScript, Css, React State',
+    github: 'nirban256',
+    cover: 'https://res.cloudinary.com/atapas/image/upload/v1650866465/demos/cover_y20bzk.png',
+    blog: '',
+    video: ''
+  },{
+    id: 'pl-quote-generator', 
+    name: 'Quote Generator', 
```

Comment:
Can we name it as "Random Quote Generator"? Just the name change.

Reply:
Name Changed!